wayshot/src/utils.rs

```diff
@@ -11,45 +11,19 @@ use std::{
 use chrono::Local;
 use libwayshot::region::{LogicalRegion, Position, Region, Size};
 
-pub fn parse_geometry(g: &str) -> Result<LogicalRegion> {
-    let tail = g.trim();
-    let x_coordinate: i32;
-    let y_coordinate: i32;
-    let width: u32;
-    let height: u32;
-
-    let validation_error =
-        "Invalid geometry provided.\nValid geometries:\n1) %d,%d %dx%d\n2) %d %d %d %d";
-
-    if tail.contains(',') {
-        // this accepts: "%d,%d %dx%d"
-        let (head, tail) = tail.split_once(',').wrap_err(validation_error)?;
-        x_coordinate = head.parse::<i32>()?;
-        let (head, tail) = tail.split_once(' ').wrap_err(validation_error)?;
-        y_coordinate = head.parse::<i32>()?;
-        let (head, tail) = tail.split_once('x').wrap_err(validation_error)?;
-        width = head.parse::<u32>()?;
-        height = tail.parse::<u32>()?;
-    } else {
-        // this accepts: "%d %d %d %d"
-        let (head, tail) = tail.split_once(' ').wrap_err(validation_error)?;
-        x_coordinate = head.parse::<i32>()?;
-        let (head, tail) = tail.split_once(' ').wrap_err(validation_error)?;
-        y_coordinate = head.parse::<i32>()?;
-        let (head, tail) = tail.split_once(' ').wrap_err(validation_error)?;
-        width = head.parse::<u32>()?;
-        height = tail.parse::<u32>()?;
+pub fn waysip_to_region(size: libwaysip::Size, point: libwaysip::Point) -> LogicalRegion {
+    let size: Size = Size {
+        width: size.width as u32,
```

Comment:
the size is u32 because the size by wl_output is i32, I think I should return the error..